doc/model/varia/atoms/axisTicksRegularity.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<c:model 
+    xmlns:c="urn:webdetails/com/2012" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="urn:webdetails/com/2012 ../../../schema/com_2012.xsd"
+    xmlns="http://www.w3.org/1999/xhtml">
+    
+    <c:atomType name="AxisTicksRegularity" space="pvc.options.varia" base="boolean">
+        <c:documentation>
```

Comment:
Don't see the need for an atom/enum type. You could merge this description with that of the property itself.

When you say:
> This only applies when the domain has some order among its elements

this is true of any continuous scale

> (some minimum and maximum values), 

what do you mean? with fixed min and max specified? because the data ultimately has a minimum and a maximum.

> and the scale domain is exactly the domain of the data.

I'm not sure whether it is mandatory or just better to have domain round mode to none. If it never makes sense, then, should we disable it altogether when this option is true?